plugins/splitbutton/plugin.js

```diff
@@ -94,12 +115,12 @@
 
 			editor.addFeature( properties.buttons[ item.id ] );
 
-			// First button as default. It might be overwritten by actual default button.
-			if ( !defaultButton && !item[ 'default' ] ) {
+			// If no default button was defined then first button will act as default.
+			if ( !defaultButton && !definition[ 'default' ] ) {
 				previousButton = properties.buttons[ item.id ];
 				defaultButton = properties.buttons[ item.id ];
 			} else {
-				if ( item[ 'default' ] ) {
+				if ( definition[ 'default' ] && definition[ 'default' ].toLowerCase() === item.name ) {
```

Comment:
`definition.default` should be given as a button name, thus it should not be case-sensitive. We expect name `Bold`, `Image` etc.